### ISSUE TITLE
How to change Table name from AspNetUsers to our own tablename

### ISSUE THREAD

**Issue by SheikPyari** [jasontaylordev/CleanArchitecture#992]
Hi Jason,
I have created the project using template

dotnet new install Clean.Architecture.Solution.Template::8.0.0-preview.6.18

and To create a ASP.NET Core Web API-only solution:

dotnet new ca-sln -cf None -o VCPAPI

when connect to  SQLDB, tables can be created automatically like 
AspNetRoleClaims
AspNetRoles
AspNetUserClaims
AspNetUserLogins
AspNetUserRoles
AspNetUsers
AspNetUserTokens

I want to change  these  table names as different names.

I tried changing this names from Infrastucture->Migrations->00000000000000_InitialCreate.cs  but not changing in DB side,

Pls help me how to change this names as different names.

Thanks, 
Sheik

**Comment by walidaslam7** [comment]
@SheikPyari 

Have you tried like this? 

```
    protected override void OnModelCreating(System.Data.Entity.DbModelBuilder modelBuilder)
    {
        base.OnModelCreating(modelBuilder);
        modelBuilder.Entity<IdentityUserLogin>().ToTable("MyUserLogins");
        modelBuilder.Entity<IdentityUserClaim>().ToTable("MyUserClaims");
        modelBuilder.Entity<IdentityRole>().ToTable("MyRoles");
    }
```

**Comment by jasontaylordev** [comment]
Please check the documentation; https://learn.microsoft.com/en-us/aspnet/core/security/authentication/customize-identity-model?view=aspnetcore-8.0#change-tablecolumn-names-and-facets.

**Comment by Scott-Atkinson** [comment]
This doesn't work as expected, I have created a new API using the clean arch command line. before building the project I remove the migrations folder and try and create a new migration, it fails.

Ran the project with the default migration and then try to change the names as per the link described, yet the project again fails 

NSwag command line tool for .NET Core Net90, toolchain v14.2.0.0 (NJsonSchema v11.1.0.0 (Newtonsoft.Json v13.0.0.0));Visit http://nswag.org/ for more information.;NSwag bin directory: C:\Users<User>.nuget\packages\nswag.msbuild\14.2.0\tools\Net90;Executing file 'config.nswag' with variables 'Configuration=Debug'...;Launcher directory: C:\Users<User>.nuget\packages\nswag.msbuild\14.2.0\tools\Net90;fail: bloodyrename.Infrastructure.Data.ApplicationDbContextInitialiser[0]; An error occurred while initialising the database.;System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.; ---> System.AggregateException: One or more errors occurred. (An error was generated for warning 'Microsoft.EntityFrameworkCore.Migrations.PendingModelChangesWarning': The model for context 'ApplicationDbContext' has pending changes. Add a new migration before updating the database. This exception can be suppressed or logged by passing event ID 'RelationalEventId.PendingModelChangesWarning' to the 'ConfigureWarnings' method in 'DbContext.OnConfiguring' or 'AddDbContext'.); ---> System.InvalidOperationException: An error was generated for warning 'Microsoft.EntityFrameworkCore.Migrations.PendingModelChangesWarning': The model for context 'ApplicationDbContext' has pending changes. Add a new migration before updating the database. This exception can be suppressed or logged by passing event ID 'RelationalEventId.PendingModelChangesWarning' to the 'ConfigureWarnings' method in 'DbContext.OnConfiguring' or 'AddDbContext'.; at Microsoft.EntityFrameworkCore.Diagnostics.EventDefinition1.Log[TLoggerCategory](IDiagnosticsLogger1 logger, TParam arg); at Microsoft.EntityFrameworkCore.Diagnostics.RelationalLoggerExtensions.PendingModelChangesWarning(IDiagnosticsLogger1 diagnostics, Type contextType); at Microsoft.EntityFrameworkCore.Migrations.Internal.Migrator.MigrateAsync(String targetMigration, CancellationToken cancellationToken); at bloodyrename.Infrastructure.Data.ApplicationDbContextInitialiser.InitialiseAsync() in C:\Dev\projects\bloodyrename\src\Infrastructure\Data\ApplicationDbContextInitialiser.cs:line 45; at bloodyrename.Infrastructure.Data.InitialiserExtensions.InitialiseDatabaseAsync(WebApplication app) in C:\Dev\projects\bloodyrename\src\Infrastructure\Data\ApplicationDbContextInitialiser.cs:line 20; at Program.<Main>$(String[] args) in C:\Dev\projects\bloodyrename\src\Web\Program.cs:line 16; at Program.<Main>(String[] args); at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor); at System.Reflection.MethodBaseInvoker.InvokeDirectByRefWithFewArgs(Object obj, Span1 copyOfArgs, BindingFlags invokeAttr); --- End of inner exception stack trace ---; at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions); at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken); at System.Threading.Tasks.Task.Wait(); at NSwag.Commands.ServiceProviderResolver.GetServiceProviderWithHostFactoryResolver(Assembly assembly) in //src/NSwag.Commands/HostApplication.cs:line 171; at NSwag.Commands.ServiceProviderResolver.GetServiceProvider(Assembly assembly) in //src/NSwag.Commands/HostApplication.cs:line 70; at NSwag.Commands.Generation.AspNetCore.AspNetCoreToOpenApiGeneratorCommandEntryPoint.Process(String commandContent, String outputFile, String applicationName) in //src/NSwag.Commands/Commands/Generation/AspNetCore/AspNetCoreToOpenApiGeneratorCommandEntryPoint.cs:line 27; at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor); at System.Reflection.MethodBaseInvoker.InvokeDirectByRefWithFewArgs(Object obj, Span1 copyOfArgs, BindingFlags invokeAttr); --- End of inner exception stack trace ---; at System.Reflection.MethodBaseInvoker.InvokeDirectByRefWithFewArgs(Object obj, Span1 copyOfArgs, BindingFlags invokeAttr); at System.Reflection.MethodBaseInvoker.InvokeWithFewArgs(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture); at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture); at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters); at NSwag.AspNetCore.Launcher.Program.Main(String[] args) in //src/NSwag.AspNetCore.Launcher/Program.cs:line 132; System.InvalidOperationException: An error was generated for warning 'Microsoft.EntityFrameworkCore.Migrations.PendingModelChangesWarning': The model for context 'ApplicationDbContext' has pending changes. Add a new migration before updating the database. This exception can be suppressed or logged by passing event ID 'RelationalEventId.PendingModelChangesWarning' to the 'ConfigureWarnings' method in 'DbContext.OnConfiguring' or 'AddDbContext'.; at Microsoft.EntityFrameworkCore.Diagnostics.EventDefinition1.Log[TLoggerCategory](IDiagnosticsLogger1 logger, TParam arg); at Microsoft.EntityFrameworkCore.Diagnostics.RelationalLoggerExtensions.PendingModelChangesWarning(IDiagnosticsLogger1 diagnostics, Type contextType); at Microsoft.EntityFrameworkCore.Migrations.Internal.Migrator.MigrateAsync(String targetMigration, CancellationToken cancellationToken); at bloodyrename.Infrastructure.Data.ApplicationDbContextInitialiser.InitialiseAsync() in C:\Dev\projects\bloodyrename\src\Infrastructure\Data\ApplicationDbContextInitialiser.cs:line 45;System.InvalidOperationException: Swagger generation failed with non-zero exit code '1'.; at NSwag.Commands.Generation.AspNetCore.AspNetCoreToOpenApiCommand.RunAsync(CommandLineProcessor processor, IConsoleHost host) in //src/NSwag.Commands/Commands/Generation/AspNetCore/AspNetCoreToOpenApiCommand.cs:line 195; at NSwag.Commands.NSwagDocumentBase.GenerateSwaggerDocumentAsync() in //src/NSwag.Commands/NSwagDocumentBase.cs:line 270; at NSwag.Commands.NSwagDocument.ExecuteAsync() in //src/NSwag.Commands/NSwagDocument.cs:line 67; at NSwag.Commands.Document.ExecuteDocumentCommand.ExecuteDocumentAsync(IConsoleHost host, String filePath) in //src/NSwag.Commands/Commands/Document/ExecuteDocumentCommand.cs:line 76; at NSwag.Commands.Document.ExecuteDocumentCommand.RunAsync(CommandLineProcessor processor, IConsoleHost host) in //src/NSwag.Commands/Commands/Document/ExecuteDocumentCommand.cs:line 33; at NConsole.CommandLineProcessor.ProcessSingleAsync(String[] args, Object input); at NConsole.CommandLineProcessor.ProcessAsync(String[] args, Object input); at NSwag.Commands.NSwagCommandProcessor.ProcessAsync(String[] args) in //src/NSwag.Commands/NSwagCommandProcessor.cs:line 65

Tried running migrations from root and from within infrastructure project, same outcome
